bin/pulumilocal

```diff
@@ -14,18 +14,20 @@ import argparse
 import subprocess
 import json
 from typing import Dict, List
+from urllib.parse import urlparse
 
 # for local testing
 PARENT_FOLDER = os.path.realpath(os.path.join(os.path.dirname(__file__), '..'))
 if os.path.isdir(os.path.join(PARENT_FOLDER, '.venv')):
     sys.path.insert(0, PARENT_FOLDER)
 
 # define global constants
-TRUE_STRINGS = ['1', 'true', 'True']
+TRUE_STRINGS = ['1', 'true']
 PULUMI_CMD = os.environ.get('PULUMI_CMD') or 'pulumi'
-LOCALSTACK_HOSTNAME = os.environ.get('LOCALSTACK_HOSTNAME') or 'localhost'
-EDGE_PORT = int(os.environ.get('EDGE_PORT') or '4566')
-USE_SSL = os.environ.get('USE_SSL') in TRUE_STRINGS
+AWS_ENDPOINT_URL = os.environ.get("AWS_ENDPOINT_URL")
+LOCALSTACK_HOSTNAME = urlparse(os.environ.get('AWS_ENDPOINT_URL')).hostname or os.environ.get("LOCALSTACK_HOSTNAME") or 'localhost'
```

Comment:
Wasn't aware that `urlparse(None)` works, but just quickly tested locally and doesn't raise an error 👍 

Small nit: would be good to have uniform style for quotes (single/double quotes) - maybe we can adjust the linter settings to use `black`/`ruff` for the `make lint` target, as we have it in our other repos (as opposed to `flake8`).

Reply:
Absolutely. In a following patch I'll add a different linter (probably ruff) and introduce pre-commit hooks, until that I fixed manually the quotation mark inconsistency.